LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ConcentratorDeduplication.cs

```diff
@@ -12,7 +12,7 @@ namespace LoRaWan.NetworkServer
     using Microsoft.Extensions.Caching.Memory;
     using Microsoft.Extensions.Logging;
 
-    public sealed class ConcentratorDeduplication :
+    public class ConcentratorDeduplication :
```

Comment:
Any reason why this is unsealed now?

Reply:
Because of the virtual method `HashKey` which needs to be virtual in order to verify calls on it. Can we avoid leaving the class unsealed?

Reply:
I wonder if we can utilize the interface we have and mock on that. But then I think we would have to make `CreateCacheKey` (both implementations) and `HashKey` members of the interface which is also not ideal (why would users of the interface need to implement these methods anyway?)

Reply:
Since we don't need the virtual method anymore for the tests we can seal the class again. I will resolve this but interested in your answer how we could avoid unsealing :)